Listener/RequestListener.php

```diff
@@ -20,33 +22,54 @@
 
 class RequestListener
 {
-    protected $ignoredRoutes;
+    /**
+     * @var array
+     */
+    private $ignoredRoutes;
 
-    protected $ignoredPaths;
+    /**
+     * @var array
+     */
+    private $ignoredPaths;
 
-    protected $newRelic;
+    /**
+     * @var Config
+     */
+    private $config;
 
-    protected $interactor;
+    /**
+     * @var NewRelicInteractorInterface
+     */
+    private $interactor;
 
-    protected $transactionNamingStrategy;
+    /**
+     * @var TransactionNamingStrategyInterface
```

Comment:
IDE are able to guess type from the constructor. Do we have to document all those parameters? If yes, what's about the class Config?

Reply:
You are correct.